devdoc/tqueue_requirements.md

```diff
@@ -273,3 +274,21 @@ TQUEUE_POP_RESULT TQUEUE_POP(T)(TQUEUE(T) tqueue, T* item, void* pop_function_co
     - **SRS_TQUEUE_01_045: [** `TQUEUE_POP(T)` shall call `dispose_item_function` with `dispose_item_function_context` as `context` and the array entry value whose state was changed to `POPPING` as `item`. **]**
 
   - **SRS_TQUEUE_01_034: [** `TQUEUE_POP(T)` shall set the state to `NOT_USED` by using `interlocked_exchange`, succeed and return `TQUEUE_POP_OK`. **]**
+
+
+### TQUEUE_GET_COUNT(T)
+```c
+int64_t TQUEUE_GET_COUNT(T)(TQUEUE(T) tqueue);
```

Comment:
>int64_t [](http://example.com/codeflow?start=0&length=7)

If the queue is created with INT32_MAX+2 elements (note, create takes a uint32_t) and there are in the queue INT32_MAX+1 elements, what does this function return?